src/app_main.c

```diff
@@ -0,0 +1,114 @@
+/*******************************************************************************
+ *   Tron Ledger Wallet
+ *   (c) 2018 Ledger
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ********************************************************************************/
+
+#include <stdbool.h>
+#include <sys/types.h>
+#include <string.h>
+
+#include "os.h"
+#include "cx.h"
+#include "os_io_seproxyhal.h"
+#include "io.h"
+#include "parser.h"
+#include "ux.h"
+
+#include "ui_idle_menu.h"
+#include "settings.h"
+#include "handlers.h"
+#include "parse.h"
+#include "app_errors.h"
+
+// The settings, stored in NVRAM.
+const internal_storage_t N_storage_real;
+
+txContent_t txContent;
+txContext_t txContext;
```

Comment:
This will lead to a lot of change that are not really necessary. I prefer not doing it as this convention (prefixing with G) is not already used in the app code